Source/Extensions/Blazorise.DataGrid/_DataGridRow.razor.cs

```diff
@@ -47,12 +47,15 @@ public override Task SetParametersAsync( ParameterView parameters )
                     case nameof( Item ):
                         Item = (TItem)parameter.Value;
                         break;
+

```

Comment:
This formating adds too much noise into commits 😅

Reply:
I noticed && reverted, I re-fixed the settings, on latest commit.

Reply:
Ignore then :)